gensim/test/test_hdpmodel.py

```diff
@@ -51,12 +52,29 @@ class TestHdpModel(unittest.TestCase, basetests.TestBaseTopicModel):
     def setUp(self):
         self.corpus = mmcorpus.MmCorpus(datapath('testcorpus.mm'))
         self.class_ = hdpmodel.HdpModel
-        self.model = self.class_(corpus, id2word=dictionary)
+        self.model = self.class_(corpus, id2word=dictionary, random_state=np.random.seed(0))
 
     def testShowTopic(self):
```

Comment:
Yeah, but that only tests the types of the results; I thought it would be important to also test for particular values. The nature of my test is different. Should I maybe name it something else?

Also, how do I make this test file run the base class tests?